pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -159,6 +169,19 @@ type ResourceBindingSpec struct {
 	// This setting applies to all Work objects created under this binding object.
 	// +optional
 	PreserveResourcesOnDeletion *bool `json:"preserveResourcesOnDeletion,omitempty"`
+
+	// Priority represents the scheduling priority of the binding.
+	// Bindings with higher priority values are scheduled ahead of bindings with lower priority values.
+	// If not specified, the priority value is set to 0.
+	// +optional
+	Priority int32 `json:"priority,omitempty"`
```

Comment:
Since we don't allow the user to override the priority in the binding, and the priority value can only be derived by the priority class, I use `int32` instead of `*int32` for now. Open to discussion.